spec/features/public_tif_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create Public object (shelved and published)', type: :feature do
+  let(:user) { create(:user) }
+  let(:user_id) { "#{user.sunet_id}@stanford.edu" }
+  let(:project_name) { "image-#{RandomWord.nouns.next}" }
+  # tif files are dark by default
+  #  see https://github.com/sul-dlss/assembly-objectfile/blob/master/lib/assembly-objectfile/content_metadata/file.rb#L9-L27
+  let(:bundle_dir) { Rails.root.join('spec/test_data/image_tif') }
+  let(:bare_druid) { 'tf111tf2222' }
+  let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'tf', '111', 'tf', '2222', bare_druid) }
+  let(:cocina_model_dark_access) { instance_double(Cocina::Models::Access, access: 'dark') }
+  let(:item) { instance_double(Cocina::Models::DRO, type: Cocina::Models::Vocab.media, access: cocina_model_dark_access) }
+  let(:dsc_object_version) { instance_double(Dor::Services::Client::ObjectVersion, openable?: true) }
+  let(:dsc_object) { instance_double(Dor::Services::Client::Object, version: dsc_object_version, find: item) }
+  let(:exp_content_md) do
+    <<~XML
+      <contentMetadata objectId="tf111tf2222" type="image">
+        <resource id="tf111tf2222_1" sequence="1" type="image">
+          <label>Image 1</label>
+          <file id="tif.tif" preserve="yes" shelve="no" publish="no">
```

Comment:
Would we expect this to be publish=“yes”?

Reply:
d'oh.  yes.  my bad.  will fix.

Reply:
fixed.